projects/core/src/lazy-loading/facade-factory/facade-factory.service.ts

```diff
@@ -119,6 +121,17 @@ export class FacadeFactoryService {
       result[property] = this.get(resolver$, property as string);
     });
 
+    result[PROXY_FACADE_INSTANCE_PROP] = true;
```

Comment:
This will add a property to each proxy facade created. And the important thing is: proxy facade created, does not mean the `ActiveCartService` is ready. This line `const resolver$ = this.getResolver(feature, facade, async);` triggers the lib loading (`this.featureModules.resolveFeature(feature)`). When lib loaded, we have something like this to replace the proxy facade.
```
{
    provide: ActiveCartFacade,
    useExisting: ActiveCartService,
 },
```
Theoretically, proxy facade created has no relationship with `ActiveCartService` is loaded.